package.json

```
@@ -74,6 +74,8 @@
     "@react-native-community/eslint-config": "^3.0.0",
     "@semantic-release/changelog": "^6.0.0",
     "@semantic-release/git": "^10.0.0",
+    "@testing-library/react-hooks": "^7.0.2",
+    "@types/jest": "^27.4.1",
```

Is 27.x necessary? We're using Jest 26.x still, and this is pulling in 27.x dependencies, potentially conflicting with existing tests.

```suggestion
    "@types/jest": "^26.0.24",
```